requirements.txt

```diff
@@ -1,4 +1,4 @@
 art>=1.8
 numpy>=1.9.0
-
+notebook>=5.2.2
```

Comment:
Why here ?!
Please add this dependency to `dev-requirements.txt`

Reply:
My fault.
It'll be handled ASAP.